gemrb/GUIScripts/LUSkillsSelection.py

```diff
@@ -89,6 +95,21 @@ def SetupSkillsWindow (pc, type, window, callback, level1=[0,0,0], level2=[1,1,1
 		SkillsNumButtons = 4
 		SkillsTextArea = SkillsWindow.GetControl (110)
 		ScrollBar = SkillsWindow.GetControl (109)
+	elif type == LUSKILLS_TYPE_LEVELUP and GameCheck.IsPST():
+		SkillsOffsetPress = -1
+		SkillsOffsetButton1 = 16
+		SkillsOffsetSum = 6
+		SkillsOffsetName = 7
+		SkillsOffsetPoints = 8
+		SkillsNumButtons = 4
+		SkillsLabelIncrement = 2
+		#The order of the buttons is the opposite of the other games
+		SkillLeftPress = SkillDecreasePress
+		SkillRightPress = SkillIncreasePress
+		#There is actually no hint text to describe the skills, so this is a dummy
+		if SkillsTextArea == 0:
```

Comment:
Why the if?

Reply:
I was trying to avoid it trying to create the text area more than once, or is that not necessary?